tests/get_packages.py

```diff
@@ -14,4 +14,4 @@
     for distribution in distributions:
         with open(f"stubs/{distribution}/METADATA.toml", "rb") as file:
             for package in tomli.load(file).get("tool", {}).get("stubtest", {}).get(METADATA_MAPPING[platform], []):
-                print(package)
+                print(f"{package} ")
```

Comment:
Why is this change needed?

Reply:
Windows-bash jank (same as above), reverted.